setup.py

```diff
@@ -89,6 +90,7 @@ def get_version(version_file):
     packages=find_packages(where="src"),
     package_dir={"": "src"},
     py_modules=[splitext(basename(path))[0] for path in glob("src/*.py")],
+    include_package_data=True,
```

Comment:
I don't think this was intended.

```suggestion
```

Reply:
This came from upstream.  Was it previously removed intentionally from this repo?